README.md

```diff
@@ -425,6 +425,7 @@ If you want to discuss the Composable Architecture or have a question about how
 
 The following translations of this README have been contributed by members of the community:
 
+* [Simplified Chinese](https://gist.github.com/sh3l6orrr/10c8f7c634a892a9c37214f3211242ad)
```

Comment:
This is awesome @sh3l6orrr, thanks for taking the time to do this! One small thing, could you alphabetize the list? Then it will be good to merge.

Reply:
Sure! I have done that.

Reply:
Oh haha, thanks for alphabetizing the whole thing! Didn't realize it had gotten out of sync. 😀

Reply:
Yeah that's why I was a little confused at the ordering 😂. It's my pleasure to contribute to TCA and the swift community😃 keep up the good work of your video series  and libraries!